card/lib/generators/card/templates/create_cards_tables.erb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
```

Comment:
This is instead of referencing a schema.rb because the app will already have its own.  So we add our tables as a migration.